README.md

```diff
@@ -104,11 +104,20 @@ This will then match +todo and log to gist1 doc.
 5. Share the doc (or docs) with your service account using the email noted above
 6. Copy content of JSON key file in to config.json
 
+### Environment Variables
+1. Open ```.env.example``` then, set all your sensetive varriables:
+```
+PRIVATE_KEY="-----BEGIN PRIVATE KEY-----\n<PRIVATE_KEY_CONTENT>\n-----END PRIVATE KEY-----\n"
+CLIENT_EMAIL="example_email@developer.gserviceaccount.com"
```

Comment:
Also let's rename `PRIVATE_KEY` and `CLIENT_EMAIL` to `GOOGLE_PRIVATE_KEY` and `GOOGLE_CLIENT_EMAIL`